core/lib/server/pub.ts

```diff
@@ -2325,3 +2325,43 @@ export const fullTextSearch = async (
 
 	return q.execute();
 };
+
+export const getExclusivelyRelatedPub = async (relatedPubId: PubsId, relationFieldSlug: string) => {
```

Comment:
what does exclusively related mean here?

Reply:
I should add a comment! An "exclusively" related pub is not something that exists yet, but I imagine in the future we may want to have an option for relation fields that ensure only one pub value exists for that relationship on a given pub. i.e. an exclusive relationship

An example of this is a parent-child hierarchy, where a child may have only one parent